### DISCUSSION TITLE
Brave package installing in inaccessible location

### DISCUSSION THREAD

**Discussion by dmorda** [chocolatey-community/chocolatey-packages#1841] [General]
I've been using chocolatey for a while on a Windows 10 host. Installing and updating packages has been working on the host and I usually do this using NinjaRMM's PowerShell running as SYSTEM to perform the choco tasks. This seems to work with the other packages from chocolatey I install, but not Brave.

I tried installing the current version of Brave and it seems that it doesn't fully install correctly.
* Not installed in C:\Program Files.
* No shortcuts added to desktop.
* Tries to install into profile location of user executing the choco command

The output from the command ```choco install brave -y``` is as follows:

```
PS C:\programdata\chocolatey\lib> choco install brave -y
Chocolatey v1.1.0
Installing the following packages:
brave
By installing, you accept licenses for the packages.
Progress: Downloading brave 1.34.81... 100%

brave v1.34.81 [Approved]
brave package files install completed. Performing other installation steps.
Checking already installed version...
Installing 64-bit brave...
brave has been installed.
brave may be able to be automatically uninstalled.
The install of brave was successful.
Software installed to 'C:\WINDOWS\system32\config\systemprofile\AppData\Local\BraveSoftware\Brave-Browser\Application'

Chocolatey installed 1/1 packages.
See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
```

I can confirm that the brave.exe file is located in the location above, but that path is not accessible to regular users. The normal Brave installer puts the application in C:\Program Files.

Does the installer need tweaking or perhaps I'm doing something wrong?

**Comment by gep13** [comment]
@pauby is this something that you are aware of?  I know that you were looking that the Brave package recently.

**Comment by pauby** [comment]
Running it from Chocolatey directly gives:

![image](https://user-images.githubusercontent.com/12760779/161778637-908c9595-8b8d-4c44-a1de-05f4f4560692.png)

The Chocolatey package runs the Brave silent installer. It doesn't specify any particular directory. 

Running the silent installer manually results in the software being installed in the same place:

![image](https://user-images.githubusercontent.com/12760779/161779681-8c4196ac-3a9c-4f64-b1f9-ec8eee4db3e5.png)

Note that the package uses the Brave SILENT installer. This is an installer that installs silently and without any user interaction and is provided directly by Brave. It's not the installer you download from the website.

**Reply by dmorda** [reply]
I noticed that Powershell is running as an administrator. Is the account you're installing it for an administrator (WDAGUtilityAxcount)?

If so, how would you install it for another user that isn't an administrator? In my setup, I have an administrator account on the computer and a standard account. I need to install it for the standard account that is a standard user.

**Reply by pauby** [reply]
Windows requires an administrative account to install software. My suggestion would be to move the files from the local user account that installed Brave, to the location you need.

**Reply by dmorda** [reply]
Yeah, that would work in this one case but wouldn't scale to install Brave across a fleet of hosts with multiple clients. This might just be a limitation with the Brave installer because other packages like Zoom or Chrome all do the right thing.

**Reply by AdmiringWorm** [reply]
This is indeed a limitation for the Brave installer (at least the silent installer that we use).

Seems there is an issue about that on their repository (https://github.com/brave/brave-browser/issues/18711).
There is also a different issue I have been following to allow a different location to be used which we could have used as well if it would be implemented.
https://github.com/brave/brave-browser/issues/598
